service/worker/migration/workflow.go

```diff
@@ -53,7 +53,10 @@ import (
 const (
 	forceReplicationWorkflowName  = "force-replication"
 	namespaceHandoverWorkflowName = "namespace-handover"
-	listExecutionPageSize         = 1000
+
+	defaultListWorkflowsPageSize = 1000
+	defaultPageCountPerExecution = 200
+	maxPageCountPerExecution     = 1000
 
 	minimumAllowedLaggingSeconds  = 5
 	minimumHandoverTimeoutSeconds = 30
```

Comment:
Not a part of this PR, but why not `time.Duration`?

Reply:
It is easier to write in JSON format just a number than time.Duration. But yes, it can be updated to use time.Duration.